winrt-related-src/schemas/appxpackage/uapmanifestschema/element-application.md

```diff
@@ -109,6 +109,8 @@ So these combinations are valid, and they all produce identical results:
 
 Similarly, if `uap10:TrustLevel="appContainer"`, then on those older systems `EntryPoint` must be set to "windows.partialTrustApplication`.
 
+Setting `uap10:TrustLevel="mediumIL"` while `uap10:RuntimeBehavior="windowsApp"` (default behavior for UWP apps) or `EntryPoint="windows.partialTrustApplication"` requires the `Microsoft.coreAppActivation_8wekyb3d8bbwe` Custom Capability, you can read more about this capability here in [Custom Capabilities](/windows/uwp/packaging/app-capability-declarations#custom-capabilities)
```

Comment:
>Setting `uap10:TrustLevel="mediumIL"` while `uap10:RuntimeBehavior="windowsApp"` (default behavior for UWP apps) or `EntryPoint="windows.partialTrustApplication"`

That's an error as TrustLevel contradicts EntryPoint. If both TL+EP (or RB+EP) are specified they must have consistent values, else error.

SUGGESTION: `Setting `uap10:TrustLevel="mediumIL"` while `uap10:RuntimeBehavior="windowsApp"` requires... This is also true if `uap10:TrustLevel="mediumIL"` and `EntryPoint` is any value other than `"windows.fullTrustApplication"` or `"windows.partialTrustApplication"`.

Reply:
I see, fixed